src/components/TableControls/TableControls.jsx

```diff
@@ -0,0 +1,132 @@
+import { useContext, useRef } from 'react'
+import { CSVLink } from 'react-csv'
+
+import styled from 'styled-components'
+import { FilterProjectsContext } from '../../context/FilterProjectsContext'
+
+import theme from '../../styles/theme'
+import { IconTrayDownload } from '../../assets/dashboardOnlyIcons'
+import { formatProjectDataHelper } from '../../helperFunctions'
+
+import { ButtonSecondary } from '../generic'
+
+const ButtonSecondaryWithMargin = styled(ButtonSecondary)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1rem;
+  padding: 0 2rem;
+`
+
+const ZeroSurveysButton = styled(ButtonSecondary)`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid ${theme.color.secondaryBorder};
+  width: 27rem;
+  cursor: pointer;
+`
+
+const StyledLabel = styled.label`
+  cursor: pointer;
+`
+
+const StyledCSVLink = styled(CSVLink)`
+  display: none;
+`
+
+const TableControlContainer = styled.div`
+  position: absolute;
+  top: 6.1rem;
+  left: 1.1rem;
+  height: 4rem;
+  z-index: 1;
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+`
+const TableControls = () => {
+  const csvLinkRef = useRef()
+
+  const {
+    checkedProjects,
+    displayedProjects,
+    selectedOrganizations,
+    setShowProjectsWithNoRecords,
+    showProjectsWithNoRecords,
+    showYourData,
+  } = useContext(FilterProjectsContext)
+
+  const tableHeaders = [
+    { label: 'Project Name', key: 'projectName' },
+    { label: 'Date Range', key: 'formattedDateRange' },
+    { label: 'Countries', key: 'countries' },
+    { label: 'Organizations', key: 'organizations' },
+    { label: 'Surveys', key: 'surveyCount' },
+    { label: 'Transects', key: 'transects' },
+  ]
+
+  const tableContent = displayedProjects
```

Comment:
I'd expect the `TableControls` component to not have the table content in it, but rather just the buttons that change things in the table. Would you consider renaming it to something like `ProjectsTable`?

Reply:
The table content inside is for the download csv, I don't think `ProjectsTable` is a good use here, since we have an actual `TableView` parent component wrapping this component.